index.html

```diff
@@ -31,17 +34,147 @@
         <h1>Hey There.<br>I’m Ricardo.</h1>
         <p class="developer">I'm a Software Developer</p>
         <p class="developer-info">I can help you build a product, feature<br>or website Look through some of my<br>work and experience! If you like what you<br>see and have a project you need coded,<br>don’t hesitate to contact me.
-        <nav>
+        <div>
           <ul class="social">
             <li><a href="https://github.com/"><img src="img/Gith.png" alt="Github"/></a></li>
             <li><a href="https://www.linkedin.com/home/?originalSubdomain=cl"><img src="img/Lnk.png" alt="Linkedin"/></a></li>
             <li><a href="https://www.instagram.com/"><img src="img/Hand.png"></a></li>
             <li><a href="https://twitter.com/?lang=es"><img src="img/Tw.png" alt="Twiter"></a></li>
             <li><a href="https://es-la.facebook.com/"><img src="img/M.png"></a></li>
           </ul>
+        </div>
+      </div>
+    </section>
+    <section class="main-info-section">
+      <div class="works-container">
+        <h2 class="recent-work">My Recent Works</h1>
+        <img src="img/Line.png" alt="" class="vector-line">
+        <img class="placeholder" src="img/Placeholder.png" class="place-image">
+        <H2 class="multi-post-h2">Multi-Post Stories</H2>
+        <p class="multi-post-p">A daily selection of privately personalized<br>reads; no accounts or sign-ups required.<br>has been the industry's standard dummy<br>text ever since the 1500s, when an<br>unknown printer took a standard dummy<br>text.</p>
+        <ul class="multi-post-ul">
+          <il><img src="img/Tagcss.png"></il>
+          <il><img src="img/Taghtml.png"></il>
+          <il><img src="img/Tagbootstrap.png"></il>
+          <il><img src="img/Tagruby.png"></il>
+        </ul>
+        <nav class="multi-post-nav">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
+      </div> 
+      <div class="profesional-art-container">
+        <h2 class="profesional-art-h2">Profesional Art<br>Printing Data</h2>
+        <p class="profesional-art-p">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
```

Comment:
- [ ] Please I suggest changing the `nav` tag to a `button` tag since it's a button. Please makes these changes to all buttons on the `index.html` file. 😄